### ISSUE TITLE
skipped level heading

### ISSUE THREAD

**Issue by r-leyshon** [banksad/returns_to_education#5]
**<h3>WCAG2.1 accessibility issues</h2>**

**Type**: Alert

**Source**: localhost
**Short desc**: "earnings by provider subject and sex" is h3 but should be h2.

**Long desc**: 

What It Means
A heading level is skipped.
Why It Matters
Headings provide document structure and facilitate keyboard navigation by users of assistive technology. These users may be confused or experience difficulty navigating when heading levels are skipped.
How to Fix It
Restructure the document headings to ensure that heading levels are not skipped. Optionally, you can use CSS to restyle the headers to any size / appearance (contrast checks would need to be compliant too though).
The Algorithm... in English
A heading level is skipped (e.g., an `<h1>` is followed by an `<h3>`, with no intermediate `<h2>`). Note that an `<h1>` is not required to be the first heading within the document.
**WCAG ref**:
* [1.3.1 Info and Relationships (Level A)](https://webaim.org/standards/wcag/checklist#sc1.3.1)
* [2.4.1 Bypass Blocks (Level A)](https://webaim.org/standards/wcag/checklist#sc2.4.1)
* [2.4.6 Headings and Labels (Level AA)](https://webaim.org/standards/wcag/checklist#sc2.4.6)

**Code ref**:

```
<h3 id="earnings-by-provider-subject-and-sex">

```
